assets/js/helpers/boot.js

```diff
@@ -76,3 +78,16 @@ $(document).ready(function() {
   });
   window.bootTopic();
 });
+
+getScript = function(src, callback) {
```

Comment:
problem is that I need to overwrite the jquery 'tooltip' that has been imported with the jquery script. for some reason, when it calls .tooltip after an ajax request the function isn't there yet I think it might be because jquery overwrote it somehow)
btw, this pull request is actually redundant. As @louisdelanoeye changed the tooltip function and    I changed it with the Jquery getScript in Rstudiovalidation of https://github.com/datacamp/rdocumentation-app/pull/149